Dockerfile

```diff
@@ -3,10 +3,12 @@ FROM node:18.18-bullseye-slim as base
 
 ARG BUILD_NUMBER=1_0_0
 ARG GIT_REF=not-available
+ARG GIT_BRANCH=main
 
 LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
 
 ENV TZ=Europe/London
+ENV GIT_BRANCH ${GIT_BRANCH}
```

Comment:
The ENV above has an `=` between key and value. I think this [needs it too](https://docs.docker.com/engine/reference/builder/#env)?